### ISSUE TITLE
[Feature Request] Support being logged in to multiple backends at once

### ISSUE THREAD

**Issue by 13hannes11** [mickael-kerjean/filestash#748]
I would like to be able to transfer files between back-ends. For example from one S3 instance to another. Currently I haven't found a way to do that.

**Comment by anakornk** [comment]
This will be a very useful. +1

**Comment by HananoshikaYomaru** [comment]
@mildred I would love to see this feature to be implemented. Every time I want to switch to another storage provider I have to logout of the current one and someone I forget the access key already and make me need to create a new access key. Also without this feature, user cannot transfer file between storage. This feature would be a very nice to have. 

I want to know how difficult is it to implement / what is the major bottlenecks of such feature?

**Comment by mickael-kerjean** [comment]
It's not trivial. Would love to see someone picking this up to create a PR but until that happen I will be closing this as nobody has shown interest in providing a PR. I will be closing this for now, if someone has interest in making it happen, feel free to comment and we'll reopen

**Comment by argrithmag** [comment]
Yeah, i have interest as well.  I'll just comment and not reopen the issue, as I don't want to spam.  But having a side by side would solve so many problems.  Right now all i can do is use filezilla in a container that implements a vnc viewer in the browser to transfer between my local nas (OMV) and my remote server.  kinda nasty.
